inference-engine/src/inference_engine/cnn_network_ngraph_impl.cpp

```diff
@@ -285,7 +290,8 @@ void CNNNetworkNGraphImpl::reshape() {
 StatusCode
 CNNNetworkNGraphImpl::reshape(const std::map<std::string, std::vector<size_t>>& inputShapes,
                         ResponseDesc* responseDesc) noexcept {
-    IE_PROFILING_AUTO_SCOPE(reshape)
+    OV_ITT_SCOPED_TASK(itt::domains::IE, "CNNNetworkNGraphImpl::reshape");
```

Comment:
I think it is better provide special macro for function tracing. From code perspective it should looks like macro without arguments in ideal. It allows to avoid cases when function was renamed, but macro still contains old name.

Reply:
Here is good example of macro overloading: https://stackoverflow.com/questions/16374776/macro-overloading
So we don't need special macro. For example we can use fallowing macro signature to generate task with function name:
```c++
OV_ITT_SCOPED_TASK(itt::domains::IE)
```

Reply:
Ok, I'll add macro without second argument.